docs/Manifesto.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: default
+title: Manifesto
+nav_order: 6
+---
+
+# DXdao Manifesto
+
+___
+
+DXdao is a decentralized collective that builds and governs decentralized products and services for the world. It aims to be an open community that reaches consensus in a transparent and fair way. 
+
+This document lays out the history, intentions and principles of the community to guide DXdao. 
+
+## Genesis 
+
+DXdao launched on Ethereum Mainnet on May 19, 2019, through a collaborative effort of the teams at Gnosis and DAOStack. Reputation (REP) holders govern DXdao. REP is not a token; it is associated with an Ethereum address and cannot be transferred. 
+
+There were 3 major ways to receive REP during the launch phase: 
+
+• 380k in lock drops rewarded for depositing ETH (80k) or other ERC20 tokens (300k) for a two week period 
+
+• 500k from trading on the DutchX exchange and registering MGN 
+
+• 100k for purchasing REP with GEN 
+
+399 Ethereum addresses participated in the initial REP distribution. Since then, one address self-slashed theirs and an additional 27 addresses have earned REP by contributing to DXdao, which has inflated total REP by 30% to 1,297,845.43 REP as of August 2020. 
+
+## DXD Fundraiser 
+
+After the initial REP distribution, DXdao moved to launch a token (DXD) to raise funds for the DAO. REP holders voted on April 25, 2020 to launch a bonding curve to mint DXD in exchange for ETH to be held in DXdao’s treasury, which is controlled by REP holders. The price to mint DXD is DXdao Manifesto denominated in ETH and goes up linearly as more DXD is minted. DXdao has a 100k DXD pre-mint that unlocks continuously over three years. The full DXdao treasury can be seen here. 
+
+DXD holders are entitled to the financial value created from DXdao products. In its current configuration, 10% of all revenue from DXdao products is deposited into the DXD smart contract buyback reserve as ETH and thus supports the price of DXD. REP holders can raise, but not lower, DXD holders’ claim to 10% of revenue. 
```

Comment:
I would also suggest to move these 2 paragraphs maybe further down the line since its not as important as the bottom text for what dxDao Manifesto is? What do you think @KeenanLukeOM ? Since they don't talk about what manifesto is....

Reply:
This is actually *technically* placeholder, since Tammy is going to work on a new manifesto. I will make some fact checks, but it should be changed before we go live regardless.